src/main/PDF/PdfController.java

```diff
@@ -91,150 +105,191 @@ public PdfController(MongoDatabase db) {
           }
         }
       };
+
+  /*
+  REQUIRES JSON Body:
+    - Body
+      - "pdfType": String giving PDF Type ("FORM", "APPLICATION", "IDENTIFICATION")
+      - if "pdfType" is "FORM"
+        - "annotated": boolean for retrieving EITHER annotated forms OR unannotated forms
+  */
   public Handler pdfGetAll =
       ctx -> {
-        String user = ctx.sessionAttribute("username");
-        String organizationName = ctx.sessionAttribute("orgName");
-        UserType privilegeLevel = ctx.sessionAttribute("privilegeLevel");
+        String user = Objects.requireNonNull(ctx.sessionAttribute("username"));
+        String organizationName = Objects.requireNonNull(ctx.sessionAttribute("orgName"));
+        UserType privilegeLevel = Objects.requireNonNull(ctx.sessionAttribute("privilegeLevel"));
+
         JSONObject req = new JSONObject(ctx.body());
-        PDFType pdfType = PDFType.createFromString(req.getString("pdfType"));
-        JSONObject res = new JSONObject();
+        PDFType pdfType =
+            Objects.requireNonNull(PDFType.createFromString(req.getString("pdfType")));
+        JSONObject res;
 
         if (pdfType == null) {
-          res.put("status", "Invalid PDFType");
+          res = PdfMessage.INVALID_PDF_TYPE.toJSON();
+        } else if (pdfType == PDFType.FORM) {
+          boolean getUnannotatedForms = Objects.requireNonNull(req.getBoolean("annotated"));
```

Comment:
Nit: This variable name does not clearly indicate that this is a boolean. Suggest: formsAreAnnotated